v2/svid/jwtsvid/svid.go

```diff
@@ -85,6 +88,11 @@ func (svid *SVID) Marshal() string {
 	return svid.token
 }
 
+// SetHint sets the hint for the JWT-SVID.
+func (svid *SVID) SetHint(hint string) {
```

Comment:
What is the benefit of the setter?

Reply:
Usually, I like to use setters for encapsulating eventual validation logic for setting the fields in the struct; especially in this case, in which this struct will and can be used by external applications. For now we don't have any validation here, and I'm not sure if we will have it in the future, so I would also be ok with not having this. What do you think?

Reply:
In this particular case, hint is an opaque string by spec, so there isn't really any validation and I don't anticipate needing it in the future.

Even so, we need to be very mindful of every exported function coming out of these packages since there is a compatibility guarantee that we make towards the stability of the interfaces. The bar should be very high towards adding new exported methods/types.